packages/docregistry/src/mimedocument.ts

```diff
@@ -147,11 +113,19 @@ class MimeDocument extends Widget implements DocumentRegistry.IReadyWidget {
   /**
    * Render the mime content.
    */
-  private _render(): Promise<void> {
+  private async _render(): Promise<void> {
```

Comment:
An aside: is TS capable of emitting the `async` keyword automatically if it knows that a promise is being returned?

Reply:
I'm not sure - TS *never* emits the async keyword for us since we are compiling for es2015, and async/await is es2016. But I doubt it will automatically emit an async keyword if one is not there in the source.